CefSharp.Core/Internals/CefTaskScheduler.h

```diff
@@ -42,7 +43,11 @@ namespace CefSharp
                 }
                 catch (Exception^)
                 {
-
+                    // This should never ever happen.
```

Comment:
Maybe we create something similar, like `Cef.UnhandledException` and raise errors (at least in the browser process, not much we can do in the sub process).